docker-bake.hcl

```diff
@@ -1,4 +1,52 @@
-target "default" {
-	output = ["type=local,dest=./build"]
+group "default" {
+  targets = ["php-8-0", "php-8-1", "php-8-2", "php-8-3", "php-8-4", "php-8-5"]
+}
+
+target "php-8-0" {
+	output = ["type=local,dest=./builds/auto/build-8.0.x"]
+	tags = ["php-wasm"]
```

Comment:
There is something wonky going on here with whitespace. Can you make it consistent (with preferably tabs)?

Reply:
Would you mind if I add an editorconfig to the repo? That should be understood by most editors, and should make fixing faster and easier; and transferrable to all other commits.

Reply:
I don't know what these are. What would that entail?

Reply:
A .editorconfig is just a file which describes for supporting IDEs; rules, which can help the IDE know tabs vs spaces and how many spaces to substitute a tab for. I'm unsure if the tabs come from the original or lack of an editorconfig led to the inconsistent spacing; but it would certainly help to make rules around files and editing explicit.